test/cluster/lwt/lwt_common.py

```diff
@@ -233,6 +274,14 @@ def select_statement(self) -> PreparedStatement:
 
     def create_workers(self, stop_event) -> List[Worker]:
         workers: List[Worker] = []
+
+        counter_stmt: Optional[PreparedStatement] = None
+        if self.use_counters:
+            counter_stmt = self.cql.prepare(
```

Comment:
But if each worker updates its own counter there would be no concurrency between updates? Better to use shared counters (`pk` is the only key), workers maintain there own deltas which in the end should match the db.